src/EditorFeatures/CSharpTest/Classification/SyntacticTaggerTests.cs

```diff
@@ -38,15 +38,12 @@ public async Task TestTagsChangedForPortionThatChanged()
 
             var checkpoint = new Checkpoint();
 
-            var notificationService = workspace.GetService<IForegroundNotificationService>();
             var tagComputer = new SyntacticClassificationTaggerProvider.TagComputer(
                 new SyntacticClassificationTaggerProvider(
                     workspace.ExportProvider.GetExportedValue<IThreadingContext>(),
-                    notificationService,
                     typeMap: null,
                     AsynchronousOperationListenerProvider.NullProvider),
                 subjectBuffer,
-                notificationService,
```

Comment:
always happen when we get less use of this type.  in an ideal world we'll remove it entirely.